test/functional/testCases.js

```diff
@@ -1785,7 +1784,7 @@ const testCases = [
                                 object_id: 'a',
                                 name: 'attr_a',
                                 type: 'Text',
-                                expression: 'a+":"+service+subservice+id+type'
```

Comment:
groupToIoTAConfigType is not providen deviceId

Maybe  testUtils.testCase should do that